packages/dts-generator/src/phases/dts-code-gen.ts

```diff
@@ -219,11 +223,15 @@ function NAMED_INTERFACE_MARKER(fqn: string): string {
  * @param ast
  * @return
  */
-function genDeprecated(ast: DeprecatedDesc) {
+function genDeprecatedOrExperimental(ast: DeprecatedDesc | ExperimentalDesc) {
   let contents = EMPTY_STRING;
-  contents += "@deprecated";
-  contents += ast.since ? ` (since ${ast.since})` : EMPTY_STRING;
-  contents += ast.description ? ` - ${ast.description}` : EMPTY_STRING;
+  if (ast.kind === "DeprecatedDesc") {
+    contents += "@deprecated";
+  } else if (ast.kind === "ExperimentalDesc") {
+    contents += "@experimental";
+  }
+  contents += ast.since ? ` as of version ${ast.since}.` : EMPTY_STRING;
```

Comment:
https://github.com/SAP/openui5/blob/master/docs/guidelines/jsdoc.md suggest a capitalized style ("As of ...")

Reply:
Thanks, done.